include/envoy/server/options.h

```diff
@@ -28,7 +28,12 @@ enum class Mode {
    */
   Validate,
 
-  // TODO(rlazarus): Add a third option for "light validation": Mock out access to the filesystem.
+  /**
+   * Completely load and initialize the config, and then exit without running the listener loop.
+   */
+  InitOnly,
```

Comment:
Is there a test for this?

Reply:
Ah no, but if #2679 can get merged it'll be way easier to add one.

Reply:
Added one....i resurrected the random component to the main_common_test base ID calculation as it seems you can't immediately open one of the required resources immediate after closing it in the previous test method.